examples/audio_classifier/raspberry_pi/README.md

```diff
@@ -0,0 +1,58 @@
+# MediaPipe Audio classifier example with Raspberry Pi
+
+This example uses [MediaPipe](https://github.com/google/mediapipe) with Python on
+a Raspberry Pi to perform real-time audio classification using audio streamed
+from the microphone.
+
+## Set up your hardware
+
+Before you begin, you need to
+[set up your Raspberry Pi](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up)
```

Comment:
I believe this only works with 64-bit Pi OS, so we should mention that as well in all of the readmes.

Reply:
I'll just post this comment once since it should apply to all of the rpi readmes.

Reply:
Alright will do. Thanks!